packages/headings/package.json

```diff
@@ -0,0 +1,72 @@
+{
+	"name": "@gov.au/headings",
+	"version": "0.1.0",
+	"description": "Class styles for display headings.",
+	"keywords": [
+		"uikit",
+		"uikit-module",
+		"scss",
+		"css",
+		"gov.au",
+		"Commonwealth of Australia",
+		"design guide",
+		"design system",
+		"headings",
+		"display",
+		"hierarchies"
+	],
+	"scripts": {
+		"postinstall": "pancake batter \"$(cd .. && npm prefix)\" -v",
+
+		"prepublish": "node ../../helper.js compile publish",
+
+		"watch:sass": "node ../../helper.js compile && onchange 'src/**/*' 'tests/site/*.scss' -- node ../../helper.js compile",
+		"serve": "browser-sync start --server --startPath 'tests/site' --files 'tests/site/*.html, tests/site/*.css, tests/site/*.js'",
+		"watch": "npm-run-all --parallel serve watch:*"
+	},
+	"dependencies": {
+		"@gov.au/pancake": "latest",
+
+		"@gov.au/core": "0.1.0"
+	},
+	"peerDependencies": {},
```

Comment:
`peerDependencies` need to have `core` in here as well. Sorry I missed that earlier